x-pack/plugins/fleet/server/constants/fleet_es_assets.ts

```diff
@@ -74,12 +74,42 @@ export const FLEET_AGENT_ID_VERIFY_COMPONENT_TEMPLATE_CONTENT = {
   },
 };
 
+export const FLEET_EVENT_INGESTED_COMPONENT_TEMPLATE_NAME = '.fleet_event_ingested-1';
+
+export const FLEET_EVENT_INGESTED_COMPONENT_TEMPLATE_CONTENT = {
+  _meta: meta,
+  template: {
+    settings: {
+      index: {
+        final_pipeline: FLEET_FINAL_PIPELINE_ID,
+      },
+    },
+    mappings: {
+      properties: {
+        event: {
```

Comment:
Should we remove the `event.ingested` mapping from `FLEET_AGENT_ID_VERIFY_COMPONENT_TEMPLATE_CONTENT`? Since both flags are true by default, it shouldn't be a breaking change to remove.